README.mediawiki

```diff
@@ -575,6 +575,46 @@ Provides linkage between OpenStackInfrastrcuture components and relevant integra
 * APIC Integration via Cisco APIC ZenPack
 * NSX Integration via NSX ZenPack
 
+
+== Daemons ==
+
+
+{| class="wikitable"
+!width="50%"|Collector Type
+!width="50%"|Daemon
+|-
+|Modeler
+|<span class="keyword cmdname">zenmodeler</span>
+|-
+|Command
+|<span class="keyword cmdname">zencommand</span>
+|-
+|OpenStack AMQP Queue Size
+|<span class="keyword cmdname">zenpython</span>
+|-
+|OpenStack Ceilometer Perf API
+|<span class="keyword cmdname">zenpython</span>
+|-
+|OpenStack Ceilometer Events AMQP
+|<span class="keyword cmdname">zenpython</span>
+|-
+|OpenStack Ceilometer Heartbeats AMQP
+|<span class="keyword cmdname">zenpython</span>
+|-
+|OpenStack Cinder Service Status
+|<span class="keyword cmdname">zenpython</span>
+|-
+|OpenStack Neutron Agent Status
+|<span class="keyword cmdname">zenpython</span>
+|-
+|OpenStack Nova Service Status
+|<span class="keyword cmdname">zenpython</span>
+|-
+|Python
```

Comment:
The intenet is to only show what daemons are being use as per the ticket, not to provice any process data.